assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -193,6 +194,49 @@ const useStoryApi = (dataAdapter, { editStoryURL, wpApi }) => {
     },
     [wpApi, dataAdapter]
   );
+  const createStoryFromTemplate = useCallback(
+    async (template) => {
+      try {
+        const { createdBy, pages, title, version } = template;
+        const copyTemplateTitle = sprintf(
+          /* translators: %s: template title */
+          __('%s (Copy)', 'web-stories'),
+          title
+        );
+
+        const storyPropsToSave = await getStoryPropsToSave({
+          story: {
+            title: copyTemplateTitle,
+            status: 'draft',
```

Comment:
This is where I think we need `status: 'auto-draft'`. But doesn't seem to be supported out of the box by the REST controller? 🤔 

As [discussed on Slack](https://xwp.slack.com/archives/C0134J07HV3/p1591214007035500), this should open the story editor pre-filled with the template. But if we create a `draft`, that pollutes the DB every time you click on `Use template`.

In other words, if you click on `Use template` 10 times, and then close all the browser tabs, no 10 stories should have been created.

cc @miina @spacedmonkey

Reply:
It seem that auto-draft is not supported is only `internal` false. 
https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/post.php#L403
https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L2110

We could add this by overriding with the custom post controller we already have. 

I am not sure what the issue is here. Creating an auto-draft, also creates a row in the database. What is the issue with it being a draft again?

Reply:
But auto-drafts are hidden to the user, and periodically deleted by WordPress.

> What is the issue with it being a draft again?

As I said: If you click on `Use template` 10 times, and then close all the browser tabs, the dashboard should not list 10 new stories. If we create a `draft`, that would be the case.

If you click `Use template`, the behavior should be the same as when you open the story for the first time. You can close the editor without your dashboard list being polluted.

Makes sense?

Reply:
That is like press create new 10 times. I think the user understand that creates a new story.

Reply:
No, the behavior is different for "Add New". Click on "Add New" 10 times. The dashboard doesn't show you 10 new stories. We want the same behavior for "Use template".

Anyway, we need to update the REST controller as you said, or find an alternative with pre-filling the content.